packages/pluggableWidgets/datagrid-web/src/utils/__tests__/useIsElementInViewport.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { renderHook } from "@testing-library/react-hooks";
+import { useIsElementInViewport } from "../useIsElementInViewport";
+
+describe("ColumnSelector useIsElementInViewport hook", () => {
+    it("Return FALSE as the element is NOT in viewport", () => {
+        const { result } = renderHook(useIsElementInViewport, {
+            initialProps: {
+                current: {
+                    // @ts-ignore
```

Comment:
Please don't use ts-ignore, it is probably never a good solution. Try casting those structures to needed types instead.

Reply:
Removed ts-ignore usage.